engine/table/src/main/java/io/deephaven/engine/table/impl/DeferredViewTable.java

```diff
@@ -289,7 +289,7 @@ protected Table redefine(TableDefinition newDefinition) {
 
     @Override
     protected Table redefine(TableDefinition newDefinitionExternal, TableDefinition newDefinitionInternal,
-            SelectColumn[] viewColumns, Map<String, Set<String>> columnDependency) {
+            SelectColumn[] viewColumns) {
```

Comment:
Let's dig into this, maybe with Nate.

Reply:
I think we can ignore. @cpwright might have context, but I can't find usages in the enterprise fork either.